### PR TITLE
gyp 0.1 (new formula)

### PR DESCRIPTION
**Pull request by c-rack**
GYP is a Meta-Build system: a build system that generates other build systems

### CODE REVIEW

File: Library/Formula/gyp.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Gyp < Formula
+  desc "Generate Your Projects"
+  homepage "https://gyp.gsrc.io/"
+  url "https://chromium.googlesource.com/external/gyp",
+    :using => :git,
+    :revision => "7d29c3de1f5eb74330b81b78ea116b67f684b23b"
```
Comment (DomT4) [inline Library/Formula/gyp.rb]: `gyp` to my knowledge doesn't tag releases, and consequently we've turned it down from the core a couple times before. I'm pretty sure this is still the case?

Comment (c-rack) [inline Library/Formula/gyp.rb]: You are right, there are still no tags. I just found some previously closed / rejected pull requests that have tried to do the same, so I guess this one might also not be merged. That's ok and my fault, as I have not searched for them before submitting the pull request.

Do you see any chance to get a formula for `gyp`? Maybe a fork of the official project that just adds tags and tarballs or similar? I am open for all ideas. Thanks!

**Comment by c-rack** [comment]
Closed as duplicate of #16333 and #11776.

File: Library/Formula/gyp.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/gyp.rb]: We'll need an official upstream tag for this to be admissible. I would be surprised if upstream ever go there, to be honest.

Nothing in the world stopping you adding this to a created tap of your own though, if you wanted to support it there. Some documentation on doing that [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md).

Comment (c-rack) [inline Library/Formula/gyp.rb]: Thanks!

**Comment by c-rack** [comment]
Just for documentation purposes, you can also install `gyp` by using this tap:
```sh
brew tap tessel/homebrew-tools
brew install gyp
```

**Pull request closed without merging**